probe/core/src/main/client/app.js

```diff
@@ -465,9 +465,16 @@ Probe.EventsRoute = Ember.Route.extend(Probe.ResetScroll, {
         if (params.page) {
             query = appendToQuery(query, 'page', params.page);
         }
-        return $.getJSON(restUrlBase + 'events' + query).done(function(data) {
-            return data;
-        }).fail(function(jqXHR, textStatus, errorThrown) {
+        return $.getJSON(restUrlBase + 'events' + query).done(
+            function(data) {
+                data.data.forEach(function(event) {
+                    event['tsShort'] = moment(event['ts']).format(
+                        'HH:mm:ss');
+                    event['tsLong'] = moment(event['ts']).format(
```

Comment:
Can a user switch to tsLong format somehow? I am thinking that having .SSS even in the short version could be useful as many events could be fired within the same second.